Assets/Scripts/Player/PlayerEntity.cs

```diff
@@ -28,78 +29,42 @@ public class PlayerEntity : MonoBehaviour
         
         private Rigidbody2D _playerRb;
         private CapsuleCollider2D _playerCollider2D;
-        private float _raycastDistance = 1.49f;
-        private float _movement;
-        private bool _isJumping;
-        
-        private void Start()
+
+        private DirectionalMover _directionalMover;
+        private Jumper _jumper;
+      
+        public void Initialize(IStatValueGiver statValueGiver)
         {
             _playerRb = GetComponent<Rigidbody2D>();
             _playerCollider2D = GetComponent<CapsuleCollider2D>();
+            
+            _directionalMover = new DirectionalMover(_playerRb,_directionalMoverData,statValueGiver);
+            _jumper = new Jumper(_playerRb, _playerCollider2D, _jumpData,statValueGiver);
             //ShowLevel();
         }
```

Comment:
Why is the method commented ? I uncommented code and it doesn't seem to work as it should. If you have a broken or unfinished feature, you shouldn't add it to the PR